fabtests/pytest/conftest.py

```diff
@@ -38,7 +38,7 @@ def pytest_addoption(parser):
                              help=option_helpmsg, default=option_default)
 
 # base ssh command
-bssh = "ssh -n -o StrictHostKeyChecking=no -o ConnectTimeout=30 -o BatchMode=yes"
+bssh = "ssh -n -o StrictHostKeyChecking=no -o ConnectTimeout=120 -o BatchMode=yes"
```

Comment:
Thanks for the PR. Do you mind explaining how come 30 seconds SSH connection timeout is not sufficient? It is unusual for the connection to take more than that.

Reply:
Hey, we are currently using 120 in other benchmarks/apps, but getting connection timeout when testing fabtests in a specific scenario
I would like to try and see if it solves timeouts. 
Not sure yet, this should be merged

Reply:
Were you able to fully mitigate the timeout with 120 seconds in your local test setup? Some verbose logging can be very helpful, e.g. `ssh -vvv`.

What particular test was impacted by this, e.g. was `ssh ConnectTimeout=30 localhost` ever timed out?